src/simulation/dynamics/DynOutput/boreAngCalc/_UnitTest/test_bore_ang_calc_inertial_heading.py

```diff
@@ -23,16 +23,18 @@
 # Author:   João Vaz Carneiro
```

Comment:
:small_red_triangle_down: Squash this commit into the previous one. Commit a new file's contents all together.

Reply:
Done